mir-module/src/main/resources/xsl/import/datacite2mods.xsl

```diff
@@ -53,16 +56,10 @@
         </mods:namePart>
     </xsl:template>
 
-    <xsl:template match="nameIdentifier[@nameIdentifierScheme='ORCID']">
+    <xsl:template match="nameIdentifier">
+        <xsl:variable name="orcid" select="mcrxml:getMatchingString(text(), $orcidRegex)"/>
```

Comment:
... and just add the new check as an additional precaution here, i.e.

```xml
<xsl:template match="nameIdentifier[@nameIdentifierScheme='ORCID']">
  <xsl:variable name="orcid" select="mcrxml:getMatchingString(text(), $orcidRegex)"/>
  <xsl:if test="string-length($orcid)!=0">
    <mods:nameIdentifier type="orcid">
      <xsl:value-of select="$orcid"/>
    </mods:nameIdentifier>
  </xsl:if>
</xsl:template>
```

Reply:
If I am doing it that way I don't get the ORCIDs imported: 10.5281/zenodo.3987859

Reply:
@toKrause I just checked for the example DOI the @nameIdentifierScheme = 'https://orcid.org' thus the template won't match